packages/analytics/src/functions.ts

```diff
@@ -137,6 +138,34 @@ export async function setUserProperties(
   }
 }
 
+/**
+ * Retrieves a unique Google Analytics identifier for the web client.
+ * See {@link https://developers.google.com/analytics/devguides/collection/ga4/reference/config#client_id | client_id}.
+ *
+ * @public
```

Comment:
Nit: I think this won't affect anything because this isn't exported anyway, but this shouldn't be tagged `@public`.